### PR TITLE
add baxter_sim launch

### PR DESCRIPTION
**Pull request by knorth55**
load jskbaxter without kiva_pod
baxter_sim

### CODE REVIEW

**Comment by knorth55** [comment]
ping @pazeshun

**Comment by pazeshun** [comment]
Sorry for late

File: jsk_2016_01_baxter_apc/launch/baxter_sim.launch [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<launch>
+  <arg name="paused" default="false"/>
+  <arg name="use_sim_time" default="true"/>
+  <arg name="gui" default="true"/>
+  <arg name="headless" default="false"/>
+  <arg name="debug" default="false"/>
+
+  <!-- We resume the logic in empty_world.launch, changing the name of the world to be launched -->
+  <include file="$(find jsk_2016_01_baxter_apc)/launch/include/baxter_sim.xml" >
+    <arg name="world_name" default="$(find baxter_gazebo)/worlds/baxter.world"/>
```
Comment (knorth55) [inline jsk_2016_01_baxter_apc/launch/baxter_sim.launch]: Oops

**Comment by knorth55** [comment]
Updated

#### small modification
- fix typo in baxter_stow_sim and baxter_pick_sim

**Pull request merged**